Sem1HW/Task2.py

```diff
@@ -0,0 +1,5 @@
+# Задача №2. Найдите сумму цифр трехзначного числа
+
+num = str(input('Введите трехзначное число: '))
```

Comment:
Функции `input` уже возвращает строковый тип данных, а значит, функция `str` тут не нужна.
```suggestion
num = input('Введите трехзначное число: ')
```